build.gradle

```diff
@@ -9,7 +9,7 @@
 buildscript {
     ext.android_support_version = '28.0.0'
     ext.kotlin_version = '1.3.0'
-    ext.android_components_version = '0.33.0'
+    ext.android_components_version = '0.34.1'
```

Comment:
From other discussions/issues, I think we really want to go to `0.35.1` or higher (latest as of 2019-01-02 should be `0.36`)

Reply:
I do understand that.  Looking through their commits, `0.34.1` had all the breaking changes that affect us.